pkgs/games/pegasus-frontend/default.nix

```
@@ -2,41 +2,37 @@
 , fetchFromGitHub
 , stdenv
 , cmake
-, qtbase
-, qtgraphicaleffects
-, qtmultimedia
-, qtsvg
-, qttools
-, qtx11extras
+, libsForQt5
 , SDL2
 , sqlite
-, wrapQtAppsHook
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "pegasus-frontend";
-  version = "unstable-2023-05-22";
+  version = "unstable-2023-12-05";
```

```suggestion
  version = "0-unstable-2023-12-05";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming